models/models.py

```diff
@@ -1,83 +1,82 @@
+from sqlalchemy import Boolean, Column, Date, Float, ForeignKey, Integer, String
 from sqlalchemy.orm import relationship
 
-from app import Base, db
+from database import Base
 
 
 class ExcludedOption(Base):
     __tablename__ = "excluded_options"
-    id = db.Column(db.Integer, primary_key=True)
-    option_type = db.Column(db.String(50), nullable=False)
-    option_value = db.Column(db.String(255), nullable=False)
+    id = Column(Integer, primary_key=True)
+    option_type = Column(String(50), nullable=False)
+    option_value = Column(String(255), nullable=False)
 
 
 class UTMLink(Base):
     __tablename__ = "utm_link"
-    id = db.Column(db.Integer, primary_key=True)
-    url = db.Column(db.String(255), nullable=False)
-    campaign_content = db.Column(db.String(50), nullable=True)
-    campaign_source = db.Column(db.String(50), nullable=False)
-    campaign_medium = db.Column(db.String(50), nullable=False)
-    campaign_name = db.Column(db.String(50), db.ForeignKey("campaign.name"), nullable=False)
-    domain = db.Column(db.String(20), nullable=False)
-    slug = db.Column(db.String(50), nullable=False)
-    short_id = db.Column(db.String(20), nullable=True)
-    short_secure_url = db.Column(db.String(20), nullable=True)
+    id = Column(Integer, primary_key=True)
+    url = Column(String(255), nullable=False)
+    campaign_content = Column(String(50), nullable=True)
+    campaign_source = Column(String(50), nullable=False)
+    campaign_medium = Column(String(50), nullable=False)
+    campaign_name = Column(String(50), ForeignKey("campaign.name"), nullable=False)
```

Comment:
Why fk on name, use classic approach to id as a primary key. It's might be a bottleneck in performance